rnd/autogpt_libs/autogpt_libs/logging/config.py

```diff
@@ -0,0 +1,199 @@
+"""Logging module for Auto-GPT."""
+
+
+import logging
+import sys
+from pathlib import Path
+
+from pydantic import Field, field_validator
+from pydantic_settings import BaseSettings, SettingsConfigDict
+from .filters import BelowLevelFilter
+from .formatters import AGPTFormatter, StructuredLoggingFormatter
+
+LOG_DIR = Path(__file__).parent.parent.parent / "logs"
+LOG_FILE = "activity.log"
+DEBUG_LOG_FILE = "debug.log"
+ERROR_LOG_FILE = "error.log"
+
+SIMPLE_LOG_FORMAT = "%(asctime)s %(levelname)s  %(title)s%(message)s"
+
+DEBUG_LOG_FORMAT = (
+    "%(asctime)s %(levelname)s %(filename)s:%(lineno)d" "  %(title)s%(message)s"
+)
+
+
+class LoggingConfig(BaseSettings):
+
+    level: str = Field(
+        default="INFO",
+        description="Logging level",
+        validation_alias="LOG_LEVEL",
+    )
+
+    enable_cloud_logging: bool = Field(
+        default=False,
+        description="Enable logging to Google Cloud Logging",
+        validation_alias="ENABLE_CLOUD_LOGGING",
+    )
+
+    enable_file_logging: bool = Field(
+        default=True,
+        description="Enable logging to file",
+        validation_alias="ENABLE_FILE_LOGGING",
+    )
+
+    enable_console_logging: bool = Field(
+        default=True,
+        description="Enable logging to console",
+        validation_alias="ENABLE_CONSOLE_LOGGING",
+    )
```

Comment:
This shouldn't be a separate option. Instead it can be derived from `ENABLE_CLOUD_LOGGING`, which is mutually exclusive with `ENABLE_CONSOLE_LOGGING`.